src/crypto/tls/common.go

```diff
@@ -466,8 +490,22 @@ const (
 	// Legacy signature and hash algorithms for TLS 1.2.
 	PKCS1WithSHA1 SignatureScheme = 0x0201
 	ECDSAWithSHA1 SignatureScheme = 0x0203
+
+	// KEMTLS algorithms for the Post-Quantum Cryptography experiment.
+	// NOTE: Do not use outside of the experiment.
+	KEMTLSWithSIKEp434 SignatureScheme = 0xfe00
+	KEMTLSWithKyber512 SignatureScheme = 0xfe01
```

Comment:
```suggestion
	KEMTLSSIKEp434 SignatureScheme = 0xfe00
	KEMTLSKyber512 SignatureScheme = 0xfe01
```
The "With" doesn't seem necessary.

Reply:
Sure!